app/controllers/bookings_controller.rb

```diff
@@ -0,0 +1,36 @@
+class BookingsController < ApplicationController
+  before_action :set_booking, only: [:show]
+  def index
+    @bookings = policy_scope(Booking).order(created_at: :desc)
+  end
+
+  def new
+    @booking = Booking.new
+    authorize @booking
+  end
+
+  def create
+    @booking = Booking.new(booking_params)
+    @booking.user = current_user
+    authorize @booking
```

Comment:
You're not setting the space either.
```suggestion
    @space = Space.find(params[:space_id])
    @booking.space = @space
    authorize @booking
```

![2019-02-14 at 15 00](https://user-images.githubusercontent.com/23463721/52768817-77231d00-3069-11e9-9c5a-2725c6bddb5e.png)